plugins/scanners/scancode/src/test/kotlin/ScanCodeResultParserTest.kt

```diff
@@ -179,3 +178,5 @@ private fun containCopyrightsExactly(vararg copyrights: Pair<String, List<TextLo
         summary.copyrightFindings.groupBy { it.statement }.entries
             .map { (key, value) -> key to value.map { it.location } }
     }
+
+private fun getAssetFile(path: String): File = File("src/test/assets", path).absoluteFile
```

Comment:
Please just import the existing function from `test-utils`. Also plugin projects have access to it.

Reply:
The existing functions only works for `funTest`.

Reply:
It shouldn't, as the `test-utils` dependency is added for *all* test suite, not just funTests:

https://github.com/oss-review-toolkit/ort/blob/521640b4f55edfbfb357f65a12db480eb9521474/buildSrc/src/main/kotlin/ort-kotlin-conventions.gradle.kts#L62

I actually tried it quickly, and it worked for me. Just the import needs to be added.

Reply:
The implementation looks like this:

```
fun getAssetFile(path: String): File = File("src/funTest/assets", path).absoluteFile
```

but the asset is under `src/test/assets`, not `src/FunTest/assets`.

Reply:
Ah, sorry, I misread your comment.